shapeshift/web
# chore: updated market function calls to match new caip19 signature

## Description

Updates function calls for market data to use caip19.

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. Check all market data in dashboard, charts and send/trade. Make sure no new errors are happening.

## Screenshots (if applicable)
